### DISCUSSION TITLE
v1.4-alpha.3

### DISCUSSION THREAD

**Discussion by arekf** [yattee/yattee#117] [Announcements]
**You are encouraged to leave your feedback in the [release discussion](https://github.com/yattee/yattee/discussions/117).**
Ideas on player controls are welcome but please _do **not** report bugs/improvements requests focusing on the current controls_ which will be rebuilt in the near future to meet needs of each supported platform.

## What's Changed
* Fix drawing player on some devices
* Improve entering full screen on macOS
* Improve handling queue
* Screen should no longer dim (iOS)/start screen saver (tvOS) while playing
* Other minor improvements

**Full Changelog**: https://github.com/yattee/yattee/compare/622dd5ba8abb64d0872412b3eaf7e4167b826cc9...v1.4-alpha.3

<hr /><em>This discussion was created from the release <a href='https://github.com/yattee/yattee/releases/tag/v1.4-alpha.3'>v1.4-alpha.3</a>.</em>

**Comment by bachig26** [comment]
will there an iPadOS version be worked in the future? (for more native compatibility)

**Reply by arekf** [reply]
iPad version has been there since day 1

**Comment by LL4ur3nz** [comment]
If i rotate to Landscape i still just get a black screen but this time if i rotate back it remains black until i close the app... (12 pro max)

**Reply by arekf** [reply]
I will investigate, thanks for reporting
